configs/seesaw_loss/cascade_mask_rcnn_r101_fpn_random_seesaw_loss_mstrain_2x_lvis_v1.py

```diff
@@ -1,6 +1,6 @@
 _base_ = [
     '../_base_/models/cascade_mask_rcnn_r50_fpn.py',
-    '../_base_/datasets/coco_instance.py',
+    '../_base_/datasets/lvis_v1_instance.py',
```

Comment:
Should not switch to `lvis_v1_instance` here because that base config uses ClassBalancedDataset to oversample the data.